setup.py

```diff
@@ -16,13 +16,13 @@
 
 setuptools.setup(
     name="multimetric",
-    version="1.3.0",
-    author="Konrad Weihmann",
-    author_email="kweihmann@outlook.com",
+    version="1.3.1",
+    author="Jason Nichols",
+    author_email="jn@ayl.us",
     description="Calculate code metrics in various languages",
     long_description=_long_description,
     long_description_content_type=_long_description_content_type,
-    url="https://github.com/priv-kweihmann/multimetric",
+    url="https://github.com/aylustltd/multimetric",
```

Comment:
Why chance the URL? - you should have permissions to push here (including rights to publish on pypi)

Reply:
Pypi release says https://github.com/startupos/multimetric

Reply:
Decided not to go ahead and merge any of that